index.html

```diff
@@ -435,7 +435,70 @@ <h4 class="languages">Skills</h4>
 
         </section>
 
-        
+        <section class="contact-me">
+            <h3 class="intro">
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </h3>
+
+            <form id="contact-form" action="https://formspree.io/f/mrgjoqgg" method="post">
+                <ul>
+                    <li>
+                        
+                        <input type="text" id="text" placeholder="Full Name" name="User_name" maxlength="30" required>
+                    </li>
+
+                    <li>
+                       <input type="email" id="email" placeholder="E-mail address" name="e-mail" required>
+                    </li>
+                    <li>
+                    
+                     <textarea id="more-info" name="more-info" placeholder="Enter your text here" maxlength="500" required></textarea>
+                    </li>
+                </ul>
+
+                <button class="button resume project-button" type="submit" >Get in touch</button>
```

Comment:
- [ ] The Button interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template. Please use the some CSS properties from the Figma design look ⬇️ 

![buttongood](https://user-images.githubusercontent.com/79058364/165333032-7d562e97-53c3-4614-a452-1f680d6fcf41.png)

Reply:
The correct button was actually implemented as on the Figma template.